tree/dataframe/inc/ROOT/RDF/RTreeColumnReader.hxx

```diff
@@ -28,6 +28,14 @@ namespace RDF {
 /// RTreeColumnReader specialization for TTree values read via TTreeReaderValues
 template <typename T>
 class R__CLING_PTRCHECK(off) RTreeColumnReader final : public ROOT::Detail::RDF::RColumnReaderBase {
+   /// The destructor resets the TTreeReaderValue object.
+   //
+   // Otherwise a race condition is present in which a TTreeReader
+   // and its TTreeReader{Value,Array}s can be deleted concurrently:
+   // - Thread #1) a task ends and pushes back processing slot
+   // - Thread #2) a task starts and overwrites thread-local TTreeReaderValues
+   // - Thread #1) first task deletes TTreeReader
+   // See https://github.com/root-project/root/commit/26e8ace6e47de6794ac9ec770c3bbff9b7f2e945
```

Comment:
Is this comment still useful? For now, I just moved it but I'm not sure it's actually still relevant with the current design of `RColumnReader`s...